include/tvm/target/tag.h

```diff
@@ -131,6 +137,11 @@ inline TargetTagRegEntry& TargetTagRegEntry::set_config(Map<String, ObjectRef> c
   return *this;
 }
 
+inline TargetTagRegEntry& TargetTagRegEntry::add_config(String key, ObjectRef value) {
```

Comment:
I think this is pretty much the same as `set_config` where you can just do
```c++
target.set_config({key, value});
```
Another example can be found  here: https://github.com/apache/tvm/blob/c3ce474dc172c521d7057061fac3b21207e83f0d/src/target/tag.cc#L73-L88

Reply:
I see. But the problem here is using set_config will **substitute** the whole config map instread of adding new items. And for macro TVM_REGISTER_CUDA_TAG, a config is already added. 

So maybe we need to find a method to add an additional config?

Reply:
> I think this is pretty much the same as `set_config` where you can just do
> 
> ```c++
> target.set_config({key, value});
> ```
> 
> Another example can be found here:
> 
> https://github.com/apache/tvm/blob/c3ce474dc172c521d7057061fac3b21207e83f0d/src/target/tag.cc#L73-L88

Thanks for the comment. But the problem here is using set_config will **substitute** the whole config map instread of adding new items. And for macro TVM_REGISTER_CUDA_TAG, a config is already added. 

So maybe we need to find a method to add an additional config?

Reply:
Thanks for the feedback, it makes sense to me.